### DISCUSSION TITLE
Instant messaging interaction

### DISCUSSION THREAD

**Discussion by nbortolotti** [nbortolotti/tflitego#19] [General]
I added Gitter community with the repository and events connected. Could be useful for updates and operations direct related to the repository.

I added a channel in Reddit for milestones and other news around the project & examples.

Open for ideas about the topic... maybe slack?

**Comment by marcoshuck** [comment]
Although I think slack is good for our instant communication, I don't see any value of having some sort of platform to chat right now.

If you want to go ahead with a instant chat group creationg, what about a `telegram` group?

**Reply by nbortolotti** [reply]
Yes agree, for now is not necessary extra load into a instant communication. Could be telegram... ok

Other related question, I know is not very popular popular but the integration with the repository updates  provided by Gitter is useful?
